apps/vaporgui/FlowEventRouter.cpp

```diff
@@ -44,6 +44,13 @@ FlowEventRouter::FlowEventRouter( QWidget *parent, ControlExec *ce)
 	qsapp->setWidgetResizable(true);
 	addTab(qsapp,"Appearance");
 
+	_seeding = new FlowSeedingSubtab(this);
+	QScrollArea* qsseed = new QScrollArea(this);
+	qsseed->setHorizontalScrollBarPolicy(Qt::ScrollBarAlwaysOff);
```

Comment:
This is something that has persisted from legacy versions of our subtabs.  Hypothetically we should not need this because all of our widgets should have a size policy that does not push the tabs beyond the width of the control panel.  However this could still occur if somebody wanted a label with a really long name.

The true fix for this would be to have a control panel that can be dynamically sized, but it's currently hard coded.  I don't know why it was done this way.  Like a lot of our legacy code, I suggest keeping this as-is until the control panel can be reassessed at a higher level.